include/internal/quic_stream_map.h

```diff
@@ -503,6 +503,40 @@ static ossl_inline ossl_unused int ossl_quic_stream_recv_get_final_size(const QU
     }
 }
 
+/*
+ * Determines the number of bytes available still to be read, and whether a FIN
+ * has yet to be read.
+ */
+static ossl_inline ossl_unused int ossl_quic_stream_recv_pending(const QUIC_STREAM *s)
+{
+    size_t avail;
+    int fin = 0;
+
+    switch (s->recv_state) {
+    default:
```

Comment:
you have all your ennumerated quic states specified as individual cases here.  Is it your intent to treat a value that is unknown (and falls into the default case), as a return 0?    Should that be an error?